keps/sig-instrumentation/2305-metrics-cardinality-enforcement/README.md

```diff
@@ -236,19 +264,145 @@ This would then be interpreted by our machinery as this:
 
 ```
 
-## Open-Question
-
+## Design Details
+### Test Plan
+Unit test to verify that the metric label will be set to "unexpected" if the metric encounters label values outside our explicit allowlist of values.
+### Graduation Criteria
+The allowlist of metrics can be configured via the exposed flag and the unit test is passed.
```

Comment:
Please fill this out for a minimum of the Alpha release (but Beta and Graduation draft plans are good to include).

Reply:
Added the For `Alpha` prefix.